ksql-common/src/main/java/io/confluent/ksql/json/KsqlTypesSerializationModule.java

```diff
@@ -16,11 +16,15 @@
 package io.confluent.ksql.json;
 
 import com.fasterxml.jackson.databind.module.SimpleModule;
+import io.confluent.ksql.schema.ksql.FormatOptions;
 import io.confluent.ksql.schema.ksql.LogicalSchema;
 
 public final class KsqlTypesSerializationModule extends SimpleModule {
-
   public KsqlTypesSerializationModule() {
-    addSerializer(LogicalSchema.class, new LogicalSchemaSerializer());
+    this(FormatOptions.none());
```

Comment:
I'd remove the default constructor so people have to make a conscious choice when to pass as the format options.